styles/all/theme/darkmode.css

```diff
@@ -1,22 +1,16 @@
-body.darkmode {
+:root.darkmode {
   filter: invert(1) hue-rotate(.5turn);
-  padding: 15px;
```

Comment:
Removed `padding: 15px` as I assume its inclusion was a mistake? Dark mode should only affect colors, not layout/spacing.